### ISSUE TITLE
What am I doing wrong? I can't get netlify to work with .org files

### ISSUE THREAD

**Issue by prosoitos** [niklasfasching/go-org#24]
While hugo renders my website written with .org files perfectly locally, I cannot get things to work at all with netlify:

- using .md files, everything works on netlify
- using .org files, the deployment is successful, but none of the code blocks, links, etc. get rendered (they are shown verbatim)

I don't think that it is a question of hugo version because I created a netlify.toml file with:

```sh
[build]
  publish = "public"
  command = "hugo"

[context.production.environment]
  HUGO_VERSION = "0.63.2"
```

*This may not be the proper place to ask about this as it likely is not a go-org problem but a netlify question... If so, don't hesitate to close this and I will ask in the netlify forum.*

**Comment by prosoitos** [comment]
I did start [a thread on the netlify forum](https://community.netlify.com/t/what-am-i-doing-wrong-my-site-built-locally-with-hugo-looks-fine-but-there-are-problems-on-netlify/8560?u=prosoitos).

**Comment by prosoitos** [comment]
I found the solution to my problem thanks to the [netlify thread](https://community.netlify.com/t/what-am-i-doing-wrong-my-site-built-locally-with-hugo-looks-fine-but-there-are-problems-on-netlify/8560/4) I opened a few days ago.

It seems that netlify is not using the latest Hugo by default and whatever version it uses by default was too old to deploy my org-mode files.

I had tried to set the Hugo version in the netlify.toml file, as advised in their docs, but for some reason, this was not having any effect.

Setting the Hugo version in the environment variables on the netlify dashboard fixed this.

So, as I suspected, it had nothing to do with go-org and I will close this issue.

Sorry for the noise, but maybe this will help someone else with the same issue some day.
